med_nex/lib/Screens/Authenticate/authenticate.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:med_nex/Screens/Authenticate/register.dart';
+import 'package:med_nex/Screens/Authenticate/sign_in.dart';
+import 'package:med_nex/Screens/Authenticate/start.dart';
+
+class Authenticate extends StatefulWidget {
+  const Authenticate({Key? key}) : super(key: key);
+
+  @override
+  _AuthenticateState createState() => _AuthenticateState();
+}
+
+class _AuthenticateState extends State<Authenticate> {
+  bool start = true;
+  bool showRegister = false;
+
+  void toggleToRegister(){
+    setState((){
+      start = false;
+      showRegister = true;
+    });
+  }
+
+  void toggleToSignIn(){
+    setState((){
+      start = false;
+      showRegister = false;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    if(start){
+      return Container(
+        child: Start(toggleToRegister: toggleToRegister, toggleToSignIn: toggleToSignIn,),
+      );
+    }
+    if(showRegister){
+      return Container(
+        child: Register(toggleToSignIn: toggleToSignIn,),
+      );
+    }
```

Comment:
This works fine. But you might want to use a switch on an enum. This would make the logic easier to read, and would make the state management a bit easier, since you would only need to change a single propery (Line 14,15; Line 19,20; line 26,27) https://stackoverflow.com/questions/56854669/how-to-switch-on-enum-in-dart